docs/10-Notes.md

```diff
@@ -85,6 +85,7 @@ There are many plugins that add additional functionality to Chart.js. Some parti
  - <a href="https://github.com/chartjs/Chart.Deferred.js" target="_blank">Chart.Deferred.js</a> - Defer initial chart update until chart scrolls into viewport
  - <a href="https://github.com/chartjs/Chart.smith.js" target="_blank">Chart.Smith.js</a> - Adds a smith chart type
  - <a href="https://github.com/chartjs/Chart.LinearGauge.js" target="_blank">Chart.LinearGauge.js</a> - Adds a linear gauge chart type
+ - <a href="https://github.com/00F100/Chart.Select2.js" target="_blank">Chart.Select2.js</a> - Use <a href="https://select2.github.io/" target="_blank">Select2</a> multiple select options do change the chart
```

Comment:
Yeap, not a plugin! :) In this repo i create only example "how to". I made a readme to explain "how to get to that point" but it is only an example. Then await the decision on where to get these "how to". I've done so much way to view and filter the graphics that will send the few examples. Tks guys!